### PR TITLE
ultraviolet lights

### PR DESCRIPTION
**Pull request by Darkmajia**
@AvianMaiden posted an image with modified uv lights for a map and i loved them so much i had to make them real. so now theyre real and anyone can make them and theyre PURPLE!!!!!!!!!!!!!!!!!!!!! theyre fucking PURPLE!!!!!!!!!!!!!!!!!

image

**Changelog**
:cl:
- add: Ultraviolet lights can now be crafted in autolathes.

### CODE REVIEW

**Comment by widgetbeck** [comment]
fix ur whitespace rq and i'll merge it

**Review by starlighthowls — APPROVED** [review]

**Comment by widgetbeck** [comment]
wolfieeeee... whitespaceeeee.......

**Comment by Darkmajia** [comment]
which whitespace?

File: Resources/Prototypes/_Impstation/Entities/Objects/Power/lights.yml
Comment (widgetbeck) [inline Resources/Prototypes/_Impstation/Entities/Objects/Power/lights.yml:1]: end of file here

**Comment by starlighthowls** [comment]
i missed it oooops

**Comment by widgetbeck** [comment]
end of file

**Comment by theqwat** [comment]
i know im WAY to late on this but i have to ask does this do anything to with plants since UV are commonly grow lights

**Comment by Darkmajia** [comment]
right now they're just dim purple lights with no extra functionality

**Pull request merged**